cob_obstacle_distance/include/cob_obstacle_distance/marker_shapes_impl.hpp

```diff
@@ -71,12 +71,14 @@ void MarkerShape<T>::init(double x, double y, double z,
     marker_.color.b = color_b;
     marker_.color.a = color_a;
 
-    marker_.header.frame_id = g_frame_id;
+    marker_.header.frame_id = root_frame;
```

Comment:
root_frame -> root_frame_id. 
As discussed transformation to root_frame is done when registering obstacles.